docusaurus.config.js

```diff
@@ -1,5 +1,5 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
```

Comment:
I love these changes.  Ideally, we should have done this as a separate PR, since it's a global change somewhat unrelated to this file.   I notice that @JerryHue has done the same thing here:

https://github.com/Seneca-ICTOER/IPC144/pull/61/files#diff-a038096cbdea434999e1dce5ab497212f1fe18204dde1a027ce3bdd663261a2a

Probably you two were discussing?  Regardless, it's a great change, and let's do it.  However, can one of you create a new PR that makes only this change, and pull it out of this PR?

Reply:
Alright, sounds good!